src/widgets/component-new-edit/components/response-format/simple/simple-date.jsx

```diff
@@ -64,7 +64,7 @@ class ResponseFormatDatatypeDate extends Component {
             </GenericOption>
 
           </Field>   
-         <div style= {{ display : formatini !== ''? 'block' : 'none' }}>
+         <div style= {{ visibility : formatini == ''? 'hidden' : 'visible' }}>
```

Comment:
I mean the hidden HTML attribute not the CSS value. 
https://caniuse.com/#feat=hidden